nb-apis/adapters-api/src/test/java/io/nosqlbench/adapters/api/templating/StrInterpolatorTest.java

```diff
@@ -85,6 +85,13 @@ public void shouldMatchNestedParens() {
         assertThat(a).isEqualTo("Uniform(0,1000000000)->int;");
     }
 
+    @Test
+    public void shouldMatchWithComments() {
+        StrInterpolator interp = new StrInterpolator(abcd);
+        String a = interp.apply("TEMPLATE(start,START)\n# TEMPLATE(blahblah,blah)\nTEMPLATE(keydist,Uniform(0,1000000000)->int);");
```

Comment:
Just a note for later, you can use the triple quote format to make multi-line strings easier to see